### DISCUSSION TITLE
Installing SSL Cert on windows version of SFTPGO

### DISCUSSION THREAD

**Discussion by pspagna** [drakkan/sftpgo#1035] [Q&A]
I am looking for some assistance installing an SSL cert on a windows version of SFTPGO.  I see a lot of documentation on linux. Has anyone ever done it successfully on windows?

Thank you

**Comment by drakkan** [comment]
Hi, it works the same way, you just need to use Windows-style paths when you set the certificate paths

**Comment by pspagna** [comment]
We tried editing the info under “httpd” in the config file with the correct paths but it doesn’t seem to work.  I can only get to the login with the :8080 even after changing it to 443 or 8443.  Still says it is unsecure.

**Reply by drakkan** [reply]
> We tried editing the info under “httpd” in the config file with the correct paths but it doesn’t seem to work. I can only get to the login with the :8080 even after changing it to 443 or 8443. Still says it is unsecure. From: Nicola Murino ***@***.***> Sent: Wednesday, October 26, 2022 12:02 PM To: drakkan/sftpgo ***@***.***> Cc: Spagna, Phil ***@***.***>; Author ***@***.***> Subject: Re: [drakkan/sftpgo] Installing SSL Cert on windows version of SFTPGO (Discussion #1035) ALERT: This email was sent from an account outside the District email system. Do not click or open attachments unless you recognize the sender and know the content is safe. Hi, it works the same way, you just need to use Windows-style paths when you set the certificate paths — Reply to this email directly, view it on GitHub<[#1035 (comment)](https://github.com/drakkan/sftpgo/discussions/1035#discussioncomment-3971655)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A32WHCO4F5G5ND6WQW4GORTWFFIXZANCNFSM6AAAAAARPENUYI>. You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
> […](#)
> ________________________________ CONFIDENTIALITY NOTICE: This electronic mail transmission is intended for the personal and confidential use of the designated recipient(s) named above. This message may contain confidential student or personnel data, an attorney-client communication, or other information otherwise protected from disclosure, and as such is privileged and confidential. If you are not the intended recipient, you are hereby notified that you have received this message and any documents in error, that any review, dissemination/disclosure, copying, distribution, or taking of any action in reliance on the contents of this information is strictly prohibited. If you have received this transmission in error, please notify the sender immediately by e-mail and delete the original message and documents. Thank you for your cooperation.

Check your logs. I only provide step-by-step support to [project sponsors/donors](https://github.com/drakkan/sftpgo#support-policy). You have to understand/fix the issue yourself, sorry
